test/fixtures/linter/autofix/ansi/016_index_error_with_jinja_if/after.sql

```diff
@@ -0,0 +1,16 @@
+-- This file combines product data from individual brands into a staging table
+{% set products =  [
+  'table1',
+  'table2'] %}
+
+{% for product in products %}
+SELECT
+    brand,
+    country_code,
+    category,
+    name,
+    id
+FROM
+{{ product }}
```

Comment:
This is wrong and shouldn't dedent. But not sure why it does. Any ideas @barrywhart ?

Reply:
I don't know exactly, but I do know that SQLFluff is conservative about applying fixes in or near templated areas (i.e. curly braces). Probably related to that. Want to create another issue for that? I'm happy to keep working on these, but trying to avoid letting any single issue become too much of a 🐇 hole.

Reply:
I confirmed just now that it indents correctly when run on a post-expansion version of the file, i.e.:
```
-- This file combines product data from individual brands into a staging table
SELECT
    brand,
    country_code,
    category,
    name,
    id
FROM
    table1
UNION ALL
SELECT
    brand,
    country_code,
    category,
    name,
    id
FROM
    table2
```